analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -118,17 +123,27 @@ private void writeMarkerFile(File rootDir) throws IOException {
 
             if (configuration.isCanBeResolved()) {
 
+                final Set<ProjectRef> allProjectDependencies = configuration.getAllDependencies()
+                        .stream()
+                        .filter(d -> ProjectDependency.class.isAssignableFrom(d.getClass()))
+                        .map(d -> new SimpleProjectRef(d.getGroup(), d.getName()))
+                        .collect(Collectors.toSet());
```

Comment:
Why not just store the project name - the String ?

Reply:
Instead of using both the group and name? 
I guess I just want to be caution and make it work no matter what. 
I could just use the name if you think it's safe.

Reply:
Ah yes good point. If we're being really safe, we should store the version as well. How about making it a set of ProjectDependencies - avoids the new SimpleProjectRef?

Reply:
We don't reallt need to check on the version do we?

Good point about `SimpleProjectRef`. But if we go down that route, then using the `Set` won't do us any good, because we won't be able to compare (via `equals`) the `Dependency` with the `UnresolvedDependency` - or perhaps you have seen something I haven't?